plugins/com.google.cloud.tools.eclipse.appengine.facets.test/src/com/google/cloud/tools/eclipse/appengine/facets/AppEngineStandardFacetTest.java

```diff
@@ -30,6 +31,11 @@
   @Mock private IRuntimeType runtimeType;
 
   @Test
+  public void testStandardFacetExists() {
+    Assert.assertTrue(ProjectFacetsManager.isProjectFacetDefined(AppEngineStandardFacet.ID));
```

Comment:
If you use the string literal here instead of the constant, then you also test the value of the constant.